src/foam/nanos/http/HttpParametersWebAgent.java

```diff
@@ -228,6 +230,8 @@ else if ( ! SafetyUtil.isEmpty(accept) && ! "application/x-www-form-urlencoded".
     }
     parameters.set("format", format);
```

Comment:
re: format override
```suggestion
    parameters.set("format", f);
```
Existing code is use   Format  = getparameters

Reply:
@jlhughes .. format is used in several other places. If I change it here to use the string instead of the enum, then I would have to change 2 other web agents (Pacs & FX) .. are you expecting this? No problem if you want it changed, just wondering if that's expected.

Reply:
so code that was using String format - was always wrong, and we should change any place that is currently using String to use Format, and only use String where we know there is a custom formatter in play.